docs/generated/devkit/CreateNodesContext.md

```diff
@@ -7,7 +7,6 @@ Context for [CreateNodesFunction](../../devkit/documents/CreateNodesFunction)
 ### Properties
 
 - [nxJsonConfiguration](../../devkit/documents/CreateNodesContext#nxjsonconfiguration)
-- [projectsConfigurations](../../devkit/documents/CreateNodesContext#projectsconfigurations)
```

Comment:
We don't have access to these any more in the area where we call createNodes, and doing so could incur a significant perf hit. I don't think its a big deal to drop this, since the API isn't documented yet and is still experimental.